linux-cachyos-hardened/PKGBUILD

```diff
@@ -608,12 +636,16 @@ _package-headers() {
     pkgdesc="Headers and scripts for building modules for the $pkgdesc kernel"
     depends=('pahole' "${pkgbase}")
 
+    if _is_lto_kernel; then
+        depends+=(clang llvm lld)
+    fi
```

Comment:
.SRCINFO doesn't respect any conditionals and will add this regardless. Since the other kernels don't default to LTO, we've decided to not add this for the other PKGBUILDs. Please see https://github.com/CachyOS/linux-cachyos/issues/346.

Reply:
Understood, although again, I thought all kernels default to LTO / Clang now, if not, would you mind explaining why not? Thanks

Reply:
https://github.com/CachyOS/linux-cachyos/issues/286. IMO it's to reduce the resources used when building kernels as the other kernel variants aren't used as much if at all compared to the default kernel. Other than that, another mass migration would need to take place by adding provides() replaces() to the PKGBUILDs temporarily which we're not quite fond of doing.